tasks/compose.yml

```diff
@@ -34,6 +34,10 @@
             - '{{ beacon_node_listening_port }}:{{ beacon_node_listening_port }}/tcp'
             - '{{ beacon_node_discovery_port }}:{{ beacon_node_discovery_port }}/udp'
           command:
+            {% if beacon_node_network == "testnet2" %}
+            - '--run'
+            - '--'
```

Comment:
What does the `--` do here?

Reply:
It separates arguments processed by `getopt` inside a shell script from those that aren't. The latter are available as "$@": https://github.com/status-im/nim-beacon-chain/blob/79f60968b93b3bee0fae75aa4a3a47dbb71e0faa/docker/shared_testnet/entry_point.sh#L66

Reply:
Okay, but what happens if `beacon_node_network` is set to `testnet1`? You don't need it then?

Reply:
No. Different Docker images with different entry points.

Reply:
Weird but okay.